docs/reference/settings/audit-settings.asciidoc

```diff
@@ -149,6 +149,14 @@ A list of authentication realm names or wildcards. The specified policy will
 not print audit events for users in these realms.
 // end::xpack-sa-lf-events-ignore-realms-tag[]
 
+[[xpack-sa-lf-events-ignore-privileges]]
+// tag::xpack-sa-lf-events-ignore-privileges-tag[]
+`xpack.security.audit.logfile.events.ignore_filters.<policy_name>.privileges`::
```

Comment:
Personally, I think the `privileges` attribute of an `ignore_filter` is not intuitive.
I would've opted for `actions`. I acknowledge that we should generally advise for the use of
privileges instead of actions, but to me the privilege is something that the user/principal has,
whereas a policy is concerned with matching over events that contain actions.
I think a `read` action over some index name, eg:
```
xpack.security.audit.logfile.events.ignore_filters.test_policy.users: ["jack"]
xpack.security.audit.logfile.events.ignore_filters.test_policy.indices: ["meta-indices-*"]
xpack.security.audit.logfile.events.ignore_filters.test_policy.actions: ["read"]
```
is clearer.
As an extreme counter example, I think:
```
xpack.security.audit.logfile.events.ignore_filters.test_policy.users: ["jack", "ad-user-*"]
xpack.security.audit.logfile.events.ignore_filters.test_policy.privileges: ["read"]
```
conveys that the policy is about users that have the `read` privilege.

We've not really discussed this, what do you think?